pyface/ui/qt4/action/status_bar_manager.py

```diff
@@ -126,4 +129,5 @@ def _show_messages(self):
         # decide to put all but the first message into separate widgets.  We
         # probably also need to extend the API to allow a "message" to be a
         # widget - depends on what wx is capable of.
-        self.status_bar.showMessage("  ".join(self.messages))
+        self.status_bar.showMessage("  ".join(self.messages),
+                                    msecs=self.message_duration)
```

Comment:
I think duration in seconds would be better.  That is what `pyface.timer` uses.

Reply:
Changed.